README.md

```diff
@@ -1,5 +1,5 @@
 <p align="center">
-<img src="https://i.imgur.com/bAZFAGF.png" width="450" height="243" alt="Kadena" title="Kadena">
+<img src="docs/img/kadena-logomark-green.png" style="width:450px; max-width:100%; height:auto;" alt="Kadena Logo" title="Kadena">
```

Comment:
Why not dropping the `style` annotation as we have it in pact-5: https://github.com/kadena-io/pact-5/blob/master/README.md?plain=1#L2  ?

Reply:
Will remove `style` annotation